### PR TITLE
fix: rename `last_login` to `last_auth_at`

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
Rename `last_login` column to `last_auth_at` for more accuracy

## Test Plan
- Check if these APIs work
  - api/v1/users should return users with `last_auth_at`
  - api/v1/user should return the current user with `last_auth_at`
- Check if `Last Seen` column display the data in user management table 

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by ashtonG — APPROVED** [review]
frontend changes good!

File: master/static/migrations/20230928101029_rename-last-login-column-in-user-table.tx.down.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE IF EXISTS public.users RENAME COLUMN last_auth_at TO last_login;
```
Comment (NicholasBlaskey) [inline master/static/migrations/20230928101029_rename-last-login-column-in-user-table.tx.down.sql:1]: we should really expect users to exist, I don't think we need the IF EXISTS

**Review by NicholasBlaskey — APPROVED** [review]
LGTM

File: master/static/migrations/20230928101029_rename-last-login-column-in-user-table.tx.down.sql (hunk 1 above)
Comment (keita-determined) [reply master/static/migrations/20230928101029_rename-last-login-column-in-user-table.tx.down.sql:1]: ok. will remove it

**Comment by keita-determined** [comment]
the `test-e2e-cpu` failure is not related to the PR, so merge this. [Slack conv](https://hpe-aiatscale.slack.com/archives/C04C9JXB1C2/p1696021740937249)

**Pull request merged**